docs/examples/go/secure_cell.go

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2020 Cossack Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+
+	"github.com/cossacklabs/themis/gothemis/cell"
+	"github.com/cossacklabs/themis/gothemis/keys"
+)
+
+func main() {
+	// Cryptographic parameters. Keep them secret.
+	key, err := keys.NewSymmetricKey()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "cannot generate symmetric key: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Input data. Note that you always encrypt bytes so strings need to be encoded.
+	// Here we just cast them into []byte, effectively encoding in UTF-8.
+	message := []byte("Broccoli or cauliflower, which one is green?")
+	context := []byte("Correct! *gunshot*")
+
+	fmt.Println("Secure Cell - Seal mode (symmetric key)")
+	// This is the easiest mode to use.
+
+	scellMK, err := cell.SealWithKey(key)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "cannot construct Secure Cell: %v\n", err)
+		os.Exit(1)
+	}
+	encrypted, err := scellMK.Encrypt(message, context)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to encrypt message: %v\n", err)
+		os.Exit(1)
+	}
+	decrypted, err := scellMK.Decrypt(encrypted, context)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to decrypt message: %v\n", err)
+		os.Exit(1)
+	}
+	fmt.Printf("Encoded:   %s\n", base64.StdEncoding.EncodeToString([]byte(message)))
+	fmt.Printf("Encrypted: %s\n", base64.StdEncoding.EncodeToString(encrypted))
+	fmt.Printf("Decrypted: %s\n", string(decrypted))
```

Comment:
Do we check somewhere, that message == decrypted?

Reply:
> Do we check somewhere, that message == decrypted?

Well, it will be obvious when an example is compiled and run. Though, it does make sense to check it programmatically and exit with a failure in case they are not equal. I'll add that in a minute.

Reply:
Awesome, thank you for these changes. I think that checking that library decrypts message into original never hurt :D